.cmake/cypher-parser.h.in

```diff
@@ -0,0 +1,6057 @@
+/* vi:set ts=4 sw=4 expandtab:
```

Comment:
Is this a duplication of `src/lib/cypher-parser.h.in`? Why is it needed?

Reply:
It is no more needed and will be deleted.

At first, I thought that, to interpolate variables from CMakeLists.txt to the final .h, the #cmakedefine instruction was required instead of a normal #define. This is false.